master/buildbot/changes/github.py

```diff
@@ -37,7 +37,7 @@
 }
 
 
-class PullRequestMixin(object):
+class PullRequestMixin():
```

Comment:
For classes which you changed to not inherit from anything, you should just remove the parentheses.